### ISSUE TITLE
test-fixture/test-fixture.html not required?

### ISSUE THREAD

**Issue by btelles** [PolymerElements/polymer-starter-kit#516]
When running gulp test:local on a clean polymer-starter-kit, the test fails on Chrome. It looks like I don't need the link/import to test-fixture/test-fixture.html? Removing that line makes tests pass on both Firefox and Chrome.

```
gulp test:local
[10:29:35] Using gulpfile ~/development/robot/gulpfile.js
[10:29:35] Starting 'wct:local'...
Installing and starting Selenium server for local browsers
Selenium server running on port 33313
Web server running on port 2000 and serving from /usr/local/home/me/robot
chrome 46                Beginning tests via http://localhost:2000/components/robot/generated-index.html?cli_browser_id=0
chrome 46                ✖ Test Suite Initialization

  Failed to execute 'registerElement' on 'Document': Registration failed for type 'test-fixture'. A type with that name is already registered.
    <unknown> at   <unknown> at /components/robot/bower_components/test-fixture/test-fixture.html:312:0
    <unknown> at   <unknown> at /components/robot/bower_components/test-fixture/test-fixture.html:315:0

chrome 46                ✖ Test Suite Initialization

  Failed to execute 'registerElement' on 'Document': Registration failed for type 'test-fixture'. A type with that name is already registered.
    <unknown> at   <unknown> at /components/robot/bower_components/test-fixture/test-fixture.html:312:0
    <unknown> at   <unknown> at /components/robot/bower_components/test-fixture/test-fixture.html:315:0

chrome 46                Tests failed: 2 failed tests
firefox 25               Beginning tests via http://localhost:2000/components/robot/generated-index.html?cli_browser_id=1
firefox 25               Tests passed
Test run ended in failure: 2 failed tests

chrome 46 (2/0/2)                       firefox 25 (2/0/0)                    
[10:29:49] 'wct:local' errored after 15 s
[10:29:49] Error: 2 failed tests
```

**Comment by mattppp** [comment]
+1. This is happening on all my wct tests.

**Comment by addyosmani** [comment]
Testing locally, I'm getting the following passing output:

```
Installing and starting Selenium server for local browsers
Selenium server running on port 65407
Web server running on port 2000 and serving from /Users/addyo/projects/io-workspace/polymer-starter-kit
safari 9.0.1             Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=3
safari 9.0.1             Tests passed
firefox 42               Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=2
chrome 47                Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=0
chrome 49                Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=1
firefox 42               Tests passed
chrome 47                Tests passed
chrome 49                Tests passed
Test run ended with great success

chrome 47 (2/0/0)
chrome 49 (2/0/0)
firefox 42 (2/0/0)
safari 9.0.1 (2/0/0)
[14:23:24] Finished 'wct:local' after 11 s
[14:23:24] Starting 'test:local'...
[14:23:24] Finished 'test:local' after 11 μs
```

This is testing against master. When you say that PSK is failing, could you confirm what version you're using and if the latest version in master (grab [here](https://github.com/PolymerElements/polymer-starter-kit/archive/v1.2.0-rc.1.zip)) still has this issue?

**Comment by mattppp** [comment]
Master works for me! It seems like test-fixture is now included with wct 3.4 so there's no reason to import it.
